.coveragerc

```diff
@@ -842,6 +842,7 @@ omit =
     homeassistant/components/pcal9535a/*
     homeassistant/components/pencom/switch.py
     homeassistant/components/philips_js/__init__.py
+    homeassistant/components/philips_js/diagnostics.py
```

Comment:
Since you do not have tests, could you add a sample to the PR description?

Reply:
Sure. Will do. It's rather big thou :)